mapper/software_management/README.md

```diff
@@ -0,0 +1,126 @@
+## Components
+
+### Thin-Edge-Json
+
+Responsibilities:
+* Define the API between a cloud provider and the DM-agent
+* Ability to convey as well:
+   * desired software lists
+   * software operation lists
+   * device profile
+   * current software lists
+   * operation status
+* Link a response to a request
+
+Out of scope:
+* Send the operation logs to the cloud.
+
+Open questions:
+* How to shorten the current software list sent by a device?
```

Comment:
Cheap solution is matching by the result of grep...

Reply:
I'm looking for a way to find the modules that has been directly installed with an install command excluding those installed as a required dependency.